mappings/net/minecraft/entity/passive/AxolotlBrain.mapping

```diff
@@ -47,20 +47,34 @@ CLASS net/minecraft/class_5768 net/minecraft/entity/passive/AxolotlBrain
 	COMMENT </table>
 	COMMENT </div>
 	FIELD field_28351 WALK_TOWARD_ADULT_RANGE Lnet/minecraft/class_6019;
+	FIELD field_30394 BREEDING_SPEED F
+	FIELD field_30395 ON_LAND_SPEED F
+	FIELD field_30396 IDLE_SPEED F
+	FIELD field_30397 TARGET_APPROACHING_SPEED F
+	FIELD field_30398 ADULT_FOLLOWING_SPEED F
 	METHOD method_33241 getTemptItems ()Lnet/minecraft/class_1856;
-	METHOD method_33242 (Lnet/minecraft/class_1309;)F
+	METHOD method_33242 getTargetApproachingSpeed (Lnet/minecraft/class_1309;)F
+		COMMENT {@return the axolotl's speed}
```

Comment:
These are speeds used by individual brain tasks, right? Should just specify that in the tag than using separate `@apiNote` sections